### DISCUSSION TITLE
JS bundle size

### DISCUSSION THREAD

**Discussion by wipfli** [protomaps/PMTiles#250] [Q&A] [answered]
How big is the javascript plugin for maplibre gl js at the moment?

If I remember correctly maplibregl is something like 700kB. If the pmtiles plugin is small enough, we could maybe incorporate it into maplibre directly instead of having it as a plugin.

**Comment by bdon** [comment, marked as answer]
zipped, less than 10kb - see badge on https://github.com/protomaps/PMTiles
